### PR TITLE
Enables port test

### PR DESCRIPTION
**Pull request by sujil02**
Adds port subcommand for containers
Updates check for no args when all flag is set.

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

File: cmd/podman/parse/common.go [hunk 1]
```diff
@@ -30,13 +30,17 @@ func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool
 		return errors.Errorf("--all and --latest cannot be used together")
 	}
 
+	if (argLen > 0) && specifiedAll {
+		return errors.Errorf("no arguments are needed with --all")
```
Comment (rhatdan) [inline cmd/podman/parse/common.go:34]: Changes this to state
"ports are not allowed with the --all option"

Comment (sujil02) [reply cmd/podman/parse/common.go:34]: This is a common check and not a specific one.

**Comment by sujil02** [comment]
@baude I have modified the check to validate no args with all flag set.
I re-ran the e2e test locally seemed fine, I could not spot a use case with args and all set together. However, if this is not the case I can revert back and add a special check-in ports

ref: https://github.com/containers/libpod/pull/6107/commits/3ada8dd3fe585c37259257bdfac94131e5a729d6#diff-c8b9edcdc2fa538036af3057366227a1R33

File: cmd/podman/parse/common.go [hunk 2]
```diff
@@ -30,13 +30,17 @@ func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool
...
+	}
+
 	if ignoreArgLen {
 		return nil
 	}
-	if (argLen > 0) && (specifiedAll || specifiedLatest) {
-		return errors.Errorf("no arguments are needed with --all or --latest")
-	} else if cidfile && (argLen > 0) && (specifiedAll || specifiedLatest || specifiedCIDFile) {
-		return errors.Errorf("no arguments are needed with --all, --latest or --cidfile")
+	if (argLen > 0) && specifiedLatest {
```
Comment (rhatdan) [inline cmd/podman/parse/common.go:40]: How about
```
             if (argLen > 0) {
                    if specifiedLatest {
		               return errors.Errorf("no arguments are needed with --latest")
	           } else  {
                      if cidfile && specifiedCIDFile {
                       		return errors.Errorf("no arguments are needed with --latest or --cidfile")
                     }
         }
```
}

Comment (sujil02) [reply cmd/podman/parse/common.go:40]: Yup. looks neater will do.
As this is a change in common error validation waiting for the test suite to run as wanted to be sure pulling the all flag validation before ignoreArgLen check looks good.

**Comment by TomSweeneyRedHat** [comment]
LGTM once @rhatdan's comment is addressed.

**Comment by sujil02** [comment]
@rhatdan  PTAL again

**Pull request merged**